Source/WebCore/platform/CaretAnimation.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2022 Apple Inc. All rights reserved.
```

Comment:
Again, copyright should match that of FrameSelection.cpp since we're copying code here.